feature-group-definitions/calc.yml

```diff
@@ -0,0 +1,8 @@
+name: CSS calc()
```

Comment:
```suggestion
name: calc() CSS function
```

Reply:
I'm not sure why I didn't call this just `calc()`.

I think the description should be "The `calc()` CSS function does thing."